runtime/oti/VMHelpers.hpp

```diff
@@ -1015,6 +1015,40 @@ class VM_VMHelpers
 		return consumed;
 	}
 
+
+	/**
+	 * Check if the UTF8 byte stream contains only ISO-8859-1/Latin-1 characters.
+	 *
+	 * @param data[in] the UTF8 byte stream
+	 * @param length[in] the length of incoming UTF8 byte stream
+	 *
+	 * @returns true if data only contains Latin-1 characters, false if otherwise
+	 */
+	static VMINLINE bool
+	isLatin1String(const U_8 *data, const UDATA length)
```

Comment:
Are we able to sue this in `j9gc_createJavaLangString` to eliminate duplicated code? I think there are other places we check for whether strings are compressible. I thought we had an API for this already as well. Might be worth checking so as to avoid duplication of code.

Reply:
This helper method is used by `j9gc_createJavaLangString`, my thought was that `VHelpers.hpp` is a better home if there is a reference from other places.

> I think there are other places we check for whether strings are compressible.

Are they `IS_STRING_COMPRESSED` and `IS_STRING_COMPRESSED_VM` which take a string object? This method expects a `U_8* data` instead.

Reply:
@fjeremic ping

Reply:
> Are they IS_STRING_COMPRESSED and IS_STRING_COMPRESSED_VM which take a string object? This method expects a U_8* data instead.

These macros only check the `coder` field. Just checked `j9gc_createJavaLangString` again. Looks ok to me.